### PR TITLE
Improve docs

### PR DESCRIPTION
**Pull request by wooorm**
This PR improves the docs in several ways.

#### ~~TODO~~DONE:

* [x] Add more badges
* [x] Proper readme’s per project
* [x] Proper main readme with sponsors etc

### CODE REVIEW

**Comment by silvenon** [comment]
You're on 🔥🔥🔥

**Comment by wooorm** [comment]
Alright, I think this is good! They’re mostly straightforward changes.
But most noteworthy are a CoC and I believe the readme changes need to be carefully reviewed: https://github.com/mdx-js/mdx/blob/6537433d6a5a6ef2fa88e120924008df00d2e5a0/readme.md.

File: packages/create-mdx/package.json [hunk 1]
```diff
@@ -1,14 +1,33 @@
...
+    "remark",
+    "unified"
+  ],
+  "homepage": "https://mdxjs.com",
+  "repository": "mdx-js/mdx",
+  "bugs": "https://github.com/mdx-js/mdx/issues",
+  "author": "John Otander <johnotander@gmail.com> (http://johnotander.com)",
+  "contributors": [
+    "John Otander <johnotander@gmail.com> (http://johnotander.com)",
+    "Tim Neutkens <timneutkens@me.com>",
```
Comment (timneutkens) [inline packages/create-mdx/package.json]: This email address is not public / should not be public 😄

```suggestion
    "Tim Neutkens <tim@zeit.co>",
```

Comment (wooorm) [reply packages/create-mdx/package.json]: Hmm, I found it somewhere, that’s why I used it! But fixed!

**Comment by timneutkens** [comment]
Awesome!

File: package.json [hunk 2]
```diff
@@ -69,7 +71,7 @@
       "git add"
     ],
     "*.md": [
-      "remark -qfo",
+      "remark -qf",
```
Comment (silvenon) [inline package.json]: Just curious, why did you remove the `-o`? If I understand correctly, that's similar to ESLint's `--fix`, it fixes what it can so in cases where all errors of a file are auto-fixable, it proceeds to `git add` and you don't have to do anything.

Comment (wooorm) [reply package.json]: Apparently it didn’t work with the `-o`. I’m not sure exactly what lint-staged is doing, but I believe it’s outputting it somehow anyway.

Comment (silvenon) [reply package.json]: 👍
